libnmstate/plugins/nmstate_plugin_ovsdb.py

```diff
@@ -169,28 +170,42 @@ def get_interfaces(self):
             else:
                 continue
 
+            # should remove external_id created by NetworkManager
+            modified_external_ids = self._drop_nm_external_ids(
```

Comment:
The comment seems to be unnecessary since it contains the same information as the method name that is called. Also at the other line.

Reply:
Done, thank you!